README.md

```diff
@@ -42,7 +42,7 @@ For more information on Apache CloudStack, please visit the [website](http://clo
 
 Apache CloudStack project uses Git. The official Git repository is at:
 
-    https://git-wip-us.apache.org/repos/asf/cloudstack.git
+    git://git.apache.org/cloudstack.git
```

Comment:
These links are 404, correct url is https://gitbox.apache.org/repos/asf/cloudstack.git